airflow/executors/lyft_celery_executor.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+
+from airflow.exceptions import AirflowException
+from airflow.executors.celery_executor import (
+    app,
+    celery_states,
+    CeleryExecutor,
+    DEFAULT_QUEUE,
+)
+from typing import Dict
+
+import json
+import logging
+import subprocess
+import os
+
+CONFIDANT_JSON_FILE = '/dev/shm/confidant/confidant_data'
```

Comment:
so whenever someone updates the confidant entry, it will rewrite this json file in this shared mount dir?

Reply:
I heard the confidant starts support roled based change confidant, but I guess it only affects on who could modify the entry in confidant ui...

Reply:
Yeah, some Confidant-daemon periodically updates this virtual file